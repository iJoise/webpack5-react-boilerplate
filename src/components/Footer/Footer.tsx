import React from "react";
import s from "./style/Footer.module.scss";

export const Footer = () => (
  <div className={s.footer}>
    <a href="https://github.com/iJoise/">GitHub</a>
    <a href="https://www.linkedin.com/in/kondratevk/">Linkedin</a>
  </div>
);
