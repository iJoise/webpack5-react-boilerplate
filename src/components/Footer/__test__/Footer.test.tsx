import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Footer } from "../Footer";

test("Footer should be render", () => {
  const { container } = render(<Footer />);

  expect(screen.getByText(/github/i)).toBeInTheDocument();
  expect(screen.getByText(/linkedin/i)).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
