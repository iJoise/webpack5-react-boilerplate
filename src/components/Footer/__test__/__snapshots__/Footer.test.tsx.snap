// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should be render 1`] = `
<div>
  <div
    class="footer"
  >
    <a
      href="https://github.com/iJoise/"
    >
      GitHub
    </a>
    <a
      href="https://www.linkedin.com/in/kondratevk/"
    >
      Linkedin
    </a>
  </div>
</div>
`;
